py/ai/h2o/sparkling/ml/algos/H2OAutoMLExtensions.py

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql.dataframe import DataFrame
+
+
+class H2OAutoMLExtensions(object):
+
+    def leaderboard(self):
```

Comment:
What is the reason for getting it separated? Just asking.... I would like to know the context

Reply:
This is done for the generation -> I wanted to extract the algo specific bits outside the algo definition to be able to generated all algos in the same way